examples/basic_operations/get_campaigns.py

```diff
@@ -45,6 +45,7 @@ def main(client, customer_id):
                 f'"{row.campaign.name}" was found.'
             )
 
+# [END get_campaigns]
```

Comment:
We usually line these up with the last line in the method, so this should look more like:

```py
    ...
    for batch in stream:
        for row in batch.results:
            print(
                f"Campaign with ID {row.campaign.id} and name "
                f'"{row.campaign.name}" was found.'
            )
            # [END get_campaigns]
```